src/scenes/home/getInvolved/successStories/successStories.css

```diff
@@ -0,0 +1,6 @@
+.successStoriesContent {
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+  justify-content: space-around;
+}
```

Comment:
This applies to all files in `src/scenes/getInvolved/successStories`:
Tthis isn't very DRY - instead of copying over the entire SuccessStories component over to 'src'/scenes/getInvolved' from 'src/scenes/landing', if that component is going to be used on two scenes, it should probabably be moved one level up to  `src/shared/components` so it can have a common reference point.

Reply:
I think it was discussed that we might remove success stories from the home page to un-clutter it a little. But I will do whatever is decided upon.